app/blueprints/newsletter.py

```
@@ -0,0 +1,286 @@
+
+        db.session.commit()
+
+        logger.info(f"Newsletter API subscription: {email or phone} (status: {status})")
+
+        return jsonify({
+            "message": "Successfully subscribed to newsletter",
+            "status": status,
+            "subscription_type": subscription_type
+        }), 200
+
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)